include/zephyr/dt-bindings/clock/stm32n6_clock.h

```diff
@@ -46,6 +46,15 @@
 #define STM32_SRC_IC19		(STM32_SRC_IC18 + 1)
 #define STM32_SRC_IC20		(STM32_SRC_IC19 + 1)
 
+#define STM32_SRC_HSI_DIV	(STM32_SRC_IC20 + 1)
+#define STM32_SRC_HCLK		(STM32_SRC_HSI_DIV + 1)
+#define STM32_SRC_I2S_CKIN	(STM32_SRC_HCLK + 1)
+#define STM32_SRC_TIMG		(STM32_SRC_I2S_CKIN + 1)
+#define STM32_SRC_PCLK1		(STM32_SRC_TIMG + 1)
+#define STM32_SRC_PCLK2		(STM32_SRC_PCLK1 + 1)
+#define STM32_SRC_PCLK4		(STM32_SRC_PCLK2 + 1)
+#define STM32_SRC_PCLK5		(STM32_SRC_PCLK4 + 1)
```

Comment:
If you add these, you should also complete the `stm32_clock_control_get_subsys_rate` function in the clock driver to be able to compute the frequency. (I2S_CKIN would require to add a new node and a new compatible, so this could be dropped. Many series have it, but none supports it at the moment)

Reply:
Keep it commented, indicating that it is not supported, like on some other series.